### PR TITLE
Add icon resources page

### PR DESCRIPTION
**Pull request by lancetharper**
Can be previewed here until this is merged: http://10.34.99.234:8080/icons/resources/

### CODE REVIEW

File: docs/icons/resources/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+**Style**
+
+These rules are subjective, and can be bent or broken.
+
+- 2px stroke weight
+- 1px radius rounded edges and terminals
+- Pixel aligned for proper anti-aliasing and rasterizing
+
+For more style guidelines, review the [Iconography Foundation](/icons/iconography/) article.
```
Comment (RatulaREI) [inline docs/icons/resources/README.md]: Should be "For more style guidelines, review the Iconography <link> article." Delete "Foundations" as it is not part of the Foundations section anymore.

File: docs/icons/resources/README.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+The Cedar Icon Library has an easy to-use [sprite creator](https://rei.github.io/cedar-icons/#/sprite).
+
+Alternately, the icon library API also provides [sprite generation](https://github.com/rei/cedar-icons#creating-a-custom-sprite).
+
+### Adding an Icon to the Cedar Icon Library
+
+1. In Abstract, create a new branch of the [CDR - Icon Contribution](https://share.goabstract.com/99335c38-51ee-41c8-8454-38c2a70c4c7f) project.
+2. Open the **Icon Contribution Area** file.
+3. Duplicate the **Template (Right-Click > Duplicate Page)** for each new icon or set of icons. (It is ok to leave alternate versions on the page, but please indicate the final version.)
+4. Follow the Iconography Foundation guidelines to ensure that your icon follows Cedar’s requirements.
```
Comment (RatulaREI) [inline docs/icons/resources/README.md]: Please delete "Foundations" and add link to the Iconography article under the newly-created Icons section.

File: docs/icons/resources/README.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+1. Download and install the Cedar SVGO Compressor Sketch plugin. **Click Assets > To Install**.
+2. Follow the usage directions for proper icon export in the plugin readme.
+3. Send your SVG file to your developer for use with the `<cdr-icon/>` component.
+
+**Note:** Icons in the vNext repository may not be in the library yet.
+
+## Questions or Issues?
+
+Contact the Cedar team using the [#cedar-user-support](https://rei.slack.com/messages/CA58YCGN4) Slack channel with your questions or issues.
```
Comment (RatulaREI) [inline docs/icons/resources/README.md]: Let's delete the "with your questions or issues" at the end. The sentence should read as: "Contact the Cedar team using the #cedar-user-support <link> Slack channel."

**Comment by lancetharper** [comment]
@RatulaREI Feel free to approve if you're satisfied now

**Pull request merged**